jetty-websocket/websocket-common/src/main/java/org/eclipse/jetty/websocket/common/io/AbstractWebSocketConnection.java

```diff
@@ -424,27 +424,34 @@ public void onFillable()
             LOG.debug("{} onFillable()", policy.getBehavior());
         }
         stats.countOnFillableEvents.incrementAndGet();
```

Comment:
This is a pointless stat.  unrelated to this issue, but dang I need to remove it.